### DISCUSSION TITLE
Handling also API registration and payments

### DISCUSSION THREAD

**Discussion by xorwen** [superfaceai/.github#14] [Ideas]
We are API provider and our API is paid.
In order to use our API via Superface, the client/developer needs to register in our system and get the API key. This is one obstacle for the client/developer.
Another obstacle buying credits to ensure the API key keeps working.

The idea is that Superface can handle both obstacles for the user by integrating both registration and payment to the API provider. Basically Superface would behave like a API reseller for some integrated providers. The client/developer would get pretty seamless experience.
Is something like this on your long-term roadmap?

**Comment by martyndavies** [comment]
Hey @xorwen, thanks for your question. Adding the account management aspect of this is not something that is on our roadmap right now.
